designer/server/src/test/scala/db/migration/V1_057__MigrateActionsAndCommentsToScenarioActivities.scala

```diff
@@ -106,7 +106,7 @@ class V1_057__MigrateActionsAndCommentsToScenarioActivities
         comment = Some(s"Very important change $expectedOldCommentIdForHeadActivity"),
         attachmentId = None,
         finishedAt = None,
-        state = Some("IN_PROGRESS"),
+        state = Some("FINISHED"),
```

Comment:
is it ok to modify the existing migration?

Reply:
Migration is not modified. It is only a test of migration. 
- it checked, that all operations/actions/comments from old tables are migrated
- it asserted, that ScenarioActivities are correctly returned for all migrated actions/comments
- but after this PR, the operation in IN_PROGRESS state would not be returned as ScenarioActivity

Reply:
oh, right :)